messages/ar.json

```diff
@@ -0,0 +1,317 @@
+      "pages-1-10-even-10-odd": "الصفحات 2 و4 و6 و8 و10 بالإضافة إلى كافة الصفحات الفردية التي يقع ترتيبها بعد صفحة 10",
+      "pages-even": "كافة الصفحات الزوجية"
+    }
+  },
+  "shuffle": {
+    "note": "ملاحظة: ادمج صفحات عدة مستندات باستخدام {link}.",
+    "help": {
+      "intro": "أداة ترتيب الصفحات من PDF Snake تعيد ترتيب الصفحات في المستند. تتيح لك هذه الأداة إمكانية إعادة ترتيب الصفحات وحذف صفحات وتكرار الصفحات وإضافة صفحات فارغة وتدوير الصفحات في خطوة واحدة.",
+      "format": "يتم إدخال ترتيب الصفحات بالتنسيق التالي؛ يشبه تنسيق نطاق الصفحات.",
+      "allThePages": "كافة صفحات المستند، بنفس الترتيب",
+      "theLastPage": "الصفحة الأخيرة في المستند",
+      "theSecondToLastPage": "من الصفحة الثانية إلى الصفحة الأخيرة في المستند",
+      "pages-15-10-3": "الصفحات 15 و10 و3",
+      "pages-15-10": "من صفحة 10 إلى صفحة 15 بترتيب معكوس",
+      "allReversed": "كافة صفحات المستند بترتيب معكوس",
+      "pages-15-10-odd": "الصفحات 15 و13 و11",
+      "pages-odd-even": "كافة الصفحات الفردية متبوعة بكافة الصفحات الزوجية",
+      "pages-odd-1-even": "كافة الصفحات الفردية متبوعة بكافة الصفحات الزوجية بترتيب معكوس",
+      "pages-5-1": "خمس نسخ من الصفحة الأولى",
+      "pages-2-odd-even-8": "كافة الصفحات الفردية، ثم كافة الصفحات الزوجية، ثم كافة الصفحات الفردية مرة أخرى، ثم كافة الصفحات الزوجية مرة أخرى، ثم صفحة 8.",
+      "pages-all-all": "نسختين من الصفحة الأولى، ثم نسختين من الصفحة الثانية، ثم نسختين من الصفحة الثالثة، إلخ.",
+      "pages-square-odd-even": "كافة الصفحات بترتيبها الطبيعي",
+      "rotateIntro": "يمكن لـ PDF Snake تدوير الصفحات أثناء إعادة ترتيبها. أدخل هذه الرموز بعد رقم الصفحة لتحديد زاوية الاستدارة:",
+      "rotate90": "استدارة 90 درجة باتجاه عقارب الساعة",
+      "rotate180": "استدارة 180 درجة",
+      "rotate270": "استدارة 270 درجة باتجاه عقارب الساعة (90 درجة عكس اتجاه عقارب الساعة)",
+      "allUpsideDown": "قلب كافة صفحات المستند من أعلى لأسفل",
+      "pages-1-2-3-4": "صفحة 1، صفحة 2 باستدارة 90 درجة باتجاه عقارب الساعة، صفحة 3 مقلوبة من أعلى إلى أسفل، صفحة 4 باستدارة 90 درجة عكس اتجاه عقارب الساعة.",
+      "groupIntro": "يتيح لك PDF Snake تكرار نمط ترتيب معين مرات كثيرة على نفس المستند. مثال:",
+      "pages-group-3": "لمستند مكون من 6 صفحات، يكون الترتيب الناتج 3 2 1 6 5 4",
+      "blankIntro": "يضيف PDF Snake صفحات فارغة عندما تدخل الحرف X: ",
+      "pages-1-X-3": "صفحة 1، صفحة فارغة، صفحة 3"
+    }
+  },
+  "pages": {
+    "zoomIn": "تكبير",
+    "zoomOut": "تصغير",
+    "download": "تنزيل الـ PDF",
+    "viewFirst": "إظهار أول {count} صفحات",
+    "viewPrevious": "إظهار الـ {count} صفحات السابقة",
+    "pageRange": "{first}-{last}",
+    "viewNext": "إظهار الـ {count} صفحات التالية",
+    "viewFinal": "إظهار آخر {count} صفحات",
+    "recalculating": "إعادة الحساب..."
+  },
+  "home": {
+    "step": "خطوة {number}: {name}",
+    "help": "مساعدة",
+    "saveAs": "حفظ هذه الخطوات باسم:",
+    "cancel": "إلغاء",
+    "saved": "تم حفظ {name}",
+    "addStep": "إضافة خطوة",
+    "saveSteps": "حفظ الخطوات"
+  },
```

Comment:
I added `saveButton` 3 days ago.  Could you add that translation here?

https://github.com/surferjeff/pdfsnake-locales/commit/a115b67ff18395c6f9d3430c17e6d9b0d74f38ec

Reply:
Done! :)